index.html

```diff
@@ -10,6 +10,7 @@ <h1>Lista de cambios</h1>
 			<li>Cambios desde request</li>
 			<li>Cambios desde request 2</li>
 			<li>Cambios directos en master</li>
+			<li>Cambios desde request-2</li>
```

Comment:
Esto es un comentario

```suggestion
			<li>Esto es una correccion</li>
```